docs/env_var.md

```diff
@@ -60,11 +60,12 @@ If you are setting multiple values for an environment variable in a list:
-|`JAVA_FONTS=<list of directories>`                         | Set this environment variable to specify the font directory. Setting this variable is equivalent to setting the property `java.awt.fonts` on Windows operating systems, and `sun.java2d.fontpath` on other operating systems. |
+|Environment&nbsp;variable                | Usage information                                                                                |
+|-----------------------------------------|--------------------------------------------------------------------------------------------------|
+|`IBM_JAVA_COMMAND_LINE`                  | This variable is set by the VM after it starts. Using this variable, you can find the command-line parameters set when the VM started. This setting is not available if the VM is invoked by using JNI.|
+|`OPENJ9_JAVA_OPTIONS=<option>`           | Set this variable to store default Java options, including **-X**, **-D**, or **-verbose:gc** style options. For example, **-Xms256m -Djava.compiler**. Any options set are overridden by equivalent options that are specified when Java is started. This variable does not support certain options, see the list in [-Xoptionsfile](xoptionsfile.md). If you specify the name of a trace output file either directly, or indirectly, by using a properties file, the output file might be accidentally overwritten if you run utilities such as the trace formatter, dump extractor, or dump viewer. To avoid this problem, add %d, %p or %t to the trace file names. See [-Xtrace:output](xtrace.md)</a>.<br/>:fontawesome-solid-pencil-alt:{: .note aria-hidden="true"} **Note:** The equivalent to `OPENJ9_JAVA_OPTIONS`, `IBM_JAVA_OPTIONS` is deprecated and will be removed in a future release.
+|`JAVA_FONTS=<list of directories>`       | Set this environment variable to specify the font directory. Setting this variable is equivalent to setting the property `java.awt.fonts` on Windows operating systems, and `sun.java2d.fontpath` on other operating systems. |
+|`_JAVA_OPTIONS=<option>`                 | Set this variable to add Java options to the end of the command line, including **-X**, **-D**, or **-verbose:gc** style options. For example, **-Xms256m -Djava.compiler**. This variable does not support certain options, see the list in [-Xoptionsfile](xoptionsfile.md). If you specify the name of a trace output file either directly, or indirectly, by using a properties file, the output file might be accidentally overwritten if you run utilities such as the trace formatter, dump extractor, or dump viewer. To avoid this problem, add %d, %p or %t to the trace file names. See [-Xtrace:output](xtrace.md)</a>.<br/>:fontawesome-solid-pencil-alt:{: .note aria-hidden="true"}
 
```

Comment:
Don't need the note icon at the end:

>` <br/>:fontawesome-solid-pencil-alt:{: .note aria-hidden="true"}`

The content could be reduced since it mostly repeats the text for OPENJ9_JAVA_OPTIONS, eg:

> Set this variable to add Java options to the end of the command line. Supported options and trace file issues are the same as for `OPENJ9_JAVA_OPTIONS=<option>`.

Reply:
Updated as suggested.